assets/css/src/newsletter_editor/components/blockTools.styl

```diff
@@ -10,41 +10,47 @@ $master-column-tool-width = 24px
 
 .mailpoet_tools
   position: absolute
-  top: 0
-  right: 0
+  bottom: 100%
+  left: 50%
   z-index: 20
-  padding: 2px
+  min-width 200px
```

Comment:
I think we should be consistent in using or not using colons in css rules. Could you please update the rules without colon?